### ISSUE TITLE
TestCli::testShow() test fails

### ISSUE THREAD

**Issue by iTzVirtual00** [keepassxreboot/keepassxc#7747]
## Overview
[NOTE]: # ( Give a BRIEF summary about your problem )
The 40th test fails at line 1965 of TestCli.cpp (keepassxc-2.7.0)

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. download the source
2.  build the source ```cmake -S keepassxc-$pkgver -B build \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DCMAKE_INSTALL_LIBDIR=lib \
    -DWITH_XC_ALL=ON \
    -DWITH_XC_UPDATECHECK=OFF
  cmake --build build```
(took from the repo on [archlinux.org](https://archlinux.org/packages/community/x86_64/keepassxc/)
4. run `xvfb-run --auto-servernum cmake --build build --target test` to run the tests (took again from the repo on [archlinux.org](https://archlinux.org/packages/community/x86_64/keepassxc/) uses that
5. wait for the tests to run

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
All tests should pass with no errors.

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
The 40th test (TestCli::testShow() in particular) fails.

## Context
[NOTE]: # ( Give us any additional information you may have. )
It seems to be just a typo of the 5th last char (15,0 != 15.0):
Actual: 
```Title: Sample Entry
UserName: User Name
Password: PROTECTED
URL: http://www.somesite.com/
Notes: Notes

Attachments:
  Sample attachment.txt (15,0 B)
                           ^
```

Expected: 
```Title: Sample Entry
UserName: User Name
Password: PROTECTED
URL: http://www.somesite.com/
Notes: Notes

Attachments:
  Sample attachment.txt (15.0 B)
                           ^
```

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.7.0
_Linux 5.16.14-1-MANJARO x86_64 GNU/Linux_

**Comment by droidmonkey** [comment]
This is because your localization is changing the delimiter.

**Comment by louib** [comment]
I'm not sure we can provide a fix for that from the tests. There doesn't seem to be a way to change the locale from within QT, although I have to say I didn't search for too long. Have a look at this answer https://stackoverflow.com/questions/13313896/changing-locale-in-qt.

@iTzVirtual00 Can you try again by prefixing the test command with `LANG=en_US.UTF-8`?

**Comment by iTzVirtual00** [comment]
> I'm not sure we can provide a fix for that from the tests. There doesn't seem to be a way to change the locale from within QT, although I have to say I didn't search for too long. Have a look at this answer https://stackoverflow.com/questions/13313896/changing-locale-in-qt.
> 
> @iTzVirtual00 Can you try again by prefixing the test command with `LANG=en_US.UTF-8`?

Sorry for the late reply. I changed the whole system number format from settings and it worked.

**Comment by oturpe** [comment]
#7780 fixes this (and other problems, too).

**Comment by dmaslenko** [comment]
@louib, @droidmonkey: if the fix #7780 was merged, does it make sense to close this issue?
